### PR TITLE
Right column out of app

### PR DESCRIPTION
**Pull request by MarieTheCoin**

### CODE REVIEW

File: apps/site-app/src/components/AppContainers/index.tsx [hunk 1]
```diff
@@ -3,17 +3,18 @@ import styles from './styles.module.less';
 
 type AppContainerProps = {
   className?: string,
+  id?: string
 }
 
 export const AppContainer : React.FC<AppContainerProps> = (props) =>
-  <div className={ `${styles.appContainer} ${props.className}` }>{props.children}</div>;
+  <div id={ `${props.id}` } className={ `${styles.appContainer} ${props.className}` }>{props.children}</div>;
```
Comment (MarieTheCoin) [inline apps/site-app/src/components/AppContainers/index.tsx:10]: Ended up not using that but will be useful later so I am leaving it

File: apps/site-app/src/containers/App/index.tsx [hunk 2]
```diff
@@ -18,8 +18,6 @@ import { MainRouter } from 'containers/MainRouter';
 import { useFxRatesStore } from '@thecointech/shared/containers/FxRate/reducer';
 import { useSidebar } from '@thecointech/shared/containers/PageSidebar/reducer';
 import { GreaterThanMobileSegment, MediaContextProvider, mediaStyles } from '@thecointech/shared/components/ResponsiveTool';
-import { ColumnRightTop } from 'containers/ColumnRight/Top';
-import { ColumnRightBottom } from 'containers/ColumnRight/Bottom';
```
Comment (MarieTheCoin) [inline apps/site-app/src/containers/App/index.tsx:22]: Those are now in the contents.

File: libs/shared/src/containers/Account/Account.tsx [hunk 3]
```diff
@@ -23,6 +23,7 @@ export type RouterPath = {
   exact?: boolean;
   icon?: SemanticICONS;
   header?: { avatar: string, primaryDescription: string, secondaryDescription: string | Element | JSX.Element };
+  key?: string
```
Comment (MarieTheCoin) [inline libs/shared/src/containers/Account/Account.tsx:26]: Will be useful leaving it.

**Review by FrozenKiwi — APPROVED** [review]
So, I understand what/why you've done it this way.

I disagree with it on a fundamental level: it very much breaks the top-down component-wise structure of the app.  It's also completely the opposite of how the sidebar functions (and they have very similar concepts).  Solving the same job in 2 different ways is the kind of things that makes future dev more difficult.

For now: bad working > good broken, and it's a philosophical issue: so lets submit this PR, but please add an issue to revisit this later.  One day, when we are less stressed, we can find a more holistic approach to this problem

**Pull request merged**